prow/github/client_test.go

```diff
@@ -2761,7 +2761,7 @@ func TestAllMethodsThatDoRequestSetOrgHeader(t *testing.T) {
 		t.Run(clientType.Method(i).Name, func(t *testing.T) {
 
 			checkingRoundTripper := testRoundTripper{func(r *http.Request) (*http.Response, error) {
-				if !strings.HasPrefix(r.URL.Path, "/app") {
+				if !strings.HasPrefix(r.URL.Path, "/app") && !strings.HasPrefix(r.URL.Path, "/user/repository_invitations") && !strings.HasPrefix(r.URL.Path, "/user/memberships/orgs") {
```

Comment:
Could we instead add the method names into the toSkip set and a small go doc (I guess a gh app can not be invited, right?)

Reply:
Moved